### ISSUE TITLE
wxRuby3: add support for persistence in wxRuby3 0.9.4

### ISSUE THREAD

**Issue by KeyWorksRW** [KeyWorksRW/wxUiEditor#1370]
### Description:

wxRuby3 just added support for persistence which should show up in version 0.9.4, so we'll need to add support for that.

**Comment by KeyWorksRW** [comment]
tldr; I'm postponing this until after the 1.2.1 release due to 0.9.7 and later of wxRuby3 not installing a binary version of wxWidgets on Windows.

Testing of wxRuby3 just got a lot more complicated for me starting with 0.9.7 because it no longer installs a binary version of wxWidgets on Windows. I'm currently testing using version 0.9.4. For anything after that I'll either need to work out the build system for Windows, or do all development/testing on WSL2 after installing Ruby/wxRuby3 on that platform (probably fedora, since the ubuntu version on WSL2 might be too old for the wxRuby3 Linux binary). It looks like about a 1.1 gig overhead to test wxRuby3 on WSL2.
